### PR TITLE
add all exempt namespaces for psc in rke1 local cluster automation

### PR DESCRIPTION
**Pull request by slickwarren**
## Issue: 

 https://github.com/rancher/rancher/issues/42719#issuecomment-1712352279

### CODE REVIEW

**Review by markusewalker — APPROVED** [review]

**Review by igomez06 — APPROVED** [review]
LGTM

File: tests/validation/tests/v3_api/resource/cluster-ha.yml [hunk 1]
```diff
@@ -28,25 +28,43 @@ services:
                enforce-version: latest
```
Comment (copyleftdev) [inline tests/validation/tests/v3_api/resource/cluster-ha.yml:28]: Your comment here.

File: tests/validation/tests/v3_api/resource/cluster-ha.yml [hunk 2]
```diff
@@ -28,25 +28,43 @@ services:
...
+               - cattle-impersonation-system
+               - cattle-istio
                - cattle-istio-system
+               - cattle-logging
                - cattle-logging-system
-               - cattle-windows-gmsa-system
+               - cattle-monitoring-system
+               - cattle-neuvector-system
+               - cattle-prometheus
+               - cattle-resources-system
```
Comment (anupama2501) [inline tests/validation/tests/v3_api/resource/cluster-ha.yml:52]: we don’t need `cattle-provisioning-capi-system`, `cattle-resources-system` to be added in the exemption list as per https://ranchermanager.docs.rancher.com/v2.7/pages-for-subheaders/rke1-hardening-guide

is there a reason why we are adding those namespaces?

Comment (slickwarren) [reply tests/validation/tests/v3_api/resource/cluster-ha.yml:52]: these were copied directly from the example here: https://ranchermanager.docs.rancher.com/reference-guides/rancher-security/psa-restricted-exemptions

Comment (slickwarren) [reply tests/validation/tests/v3_api/resource/cluster-ha.yml:52]: capi was added after debugging an additional problem found as part of this ticket, and will be added to the docs

**Pull request merged**